src/FoodStorage/foodStorage.py

```diff
@@ -3,13 +3,39 @@
 class FoodStorage:
     def __init__(self):
         self.isOpen = False
-        self.__inventory__ = PriorityQueue()
+        self._inventoryPQ = PriorityQueue()
+        self._sortedInventory = []
 
+    ''' Adds each food item from list_of_foods to self._inventoryPQ '''
+    def addFoods(self, list_of_foods):
+        for food in list_of_foods:
+            self.add(food)
+        print("Successfully added all foods: ${list_of_foods}")
+
+    ''' Adds food item to self._inventoryPQ. '''
     def add(self, food):
-        return
+        date = food.expiration_date
+        self._inventoryPQ.put((date, food))
 
     def remove(self, food):
         return
+
+    ''' Updates self._sortedInventory. If there were new foods added, since we last 
+        updated our self._sortedInventory, we add everything from the current 
+        self._sortedInventory into self._inventoryPQ and use it to repopulate 
+        self._sortedInventory_. '''
+    def _sortInventory(self):
+        if not self._inventoryPQ.empty():
+            while self._sortedInventory:
+                self._inventoryPQ.put(self._sortedInventory.pop())
+
+            while not self._inventoryPQ.empty():
+                self._sortedInventory.append(self._inventoryPQ.get())
+
+    ''' Lists our inventory in order of earliest expiration date. '''
+    def list(self):
+        self.__sortInventory__()
+        return self._sortedInventory
     
```

Comment:
Does 'def _sortInventory' work as an update function, or not because it doesn't take into account if something is removed from the inventory?

Reply:
It basically updates `self._sortedInventory`. For right now, since we're only implementing `add`, it'll only update `self._sortedInventory` when new foods are added. Later, it will be expanded for `remove`, `update` and whatever we see fit.